dynamiqs/time_tensor.py

```diff
@@ -57,14 +59,55 @@ def totime(
                 f' a square matrix, but has shape {tuple(tensor.shape)}.'
             )
 
-        factors = [_PWCTensor(times, values)]
+        factors = [_PWCFactor(times, values)]
         tensors = tensor.unsqueeze(0)  # (1, n, n)
         return PWCTimeTensor(factors, tensors)
-    # constant time tensor
+    # === modulated time tensor
+    if isinstance(x, tuple) and len(x) == 2:
+        factor, tensor = x
+
+        # check factor
+        if not callable(factor):
+            raise TypeError(
+                'For a modulated time tensor `(factor, tensor)`, argument `factor` must'
+                f' be a callable, but has type {obj_type_str(factor)}.'
```

Comment:
Can you replace "callable" by "function" in all errors message? (more accessible terminology)